certbot-apache/certbot_apache/tests/centos6_test.py

```diff
@@ -95,6 +94,26 @@ def test_loadmod_default(self):
             ifmod_args = self.config.parser.get_all_args(lm[:-17])
             self.assertTrue("!mod_ssl.c" in ifmod_args)
 
+    def test_loadmod_multiple(self):
+        sslmod_args = ["ssl_module", "modules/mod_ssl.so"]
+        # Adds another LoadModule to main httpd.conf in addtition to ssl.conf
+        self.config.parser.add_dir(self.config.parser.loc["default"], "LoadModule",
+                                   sslmod_args)
+        self.config.save()
+        pre_loadmods = self.config.parser.find_dir(
+            "LoadModule", "ssl_module", exclude=False)
+        # LoadModules are not within IfModule blocks
+        self.assertFalse(any(["ifmodule" in m.lower() for m in pre_loadmods]))
+        self.config.assoc["test.example.com"] = self.vh_truth[0]
+        self.config.deploy_cert(
+            "random.demo", "example/cert.pem", "example/key.pem",
+            "example/cert_chain.pem", "example/fullchain.pem")
+        post_loadmods = self.config.parser.find_dir(
+            "LoadModule", "ssl_module", exclude=False)
+
+        for mod in post_loadmods:
+            self.assertTrue(self.config.parser.not_modssl_ifmodule(mod))  #pylint: disable=no-member
```

Comment:
Did you check if adding a line before this like:
```
assert isinstance(self.config.parser, CentOSParser)
```
satisfies `pylint`?

Reply:
Unfortunately it doesn't :/